docs/features/tenancy.md

```diff
@@ -0,0 +1,13 @@
+# Tenancy
+
+Multi-tenancy is implemented via assigning VMs and Datastore to a tenant. A tenant can be granted access to create, delete or mount volumes on a specific datastore. VMs assigned to a tenant can then execute Docker volume APIs on an assigned datastores. Within a datastore multiple tenants can store their Docker volumes. A tenant cannot access volumes created by a different tenant, i.e. tenants have their own independent namespace even if tenants share datastores. VMs cannot be shared between tenants.
+
+## Admin CLI
+
+Tenants can be created and managed via the [Admin
+CLI](/user-guide/admin-cli/#tenant)
+
+## References
+
+- [Design Spec for tenancy](https://github.com/vmware/docker-volume-vsphere/blob/master/docs/misc/docker-volume-auth-proposal.v1_2.md)
+- [Introduction to vSphere Storage](https://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.storage.doc/GUID-F602EB17-8D24-400A-9B05-196CEA66464F.html)
```

Comment:
We are introducing terms such as datastore without giving any background.. the design spec is public and a good read, we can skip that.